packages/protocol/test/governance/validators.ts

```diff
@@ -594,35 +595,21 @@ contract('Validators', (accounts: string[]) => {
           event: 'ValidatorRegistered',
           args: {
             validator,
-            name,
             publicKeysData,
           },
         })
       })
     })
 
-    describe('when the account is already a registered validator', () => {
```

Comment:
Why remove this test?

Reply:
I removed it because there wasn't any test in this describe group and it looks like a dupe from the describe group below, at least judging from the description. But not I'm realizing that the group below was probably just a typo, so I readded to what I think it was meant to test (reverts of in case of prior registration of validators/validator groups)

Reply:
Yeah probably just a mistake on my part, thanks for cleaning up after me 😄